src/cpp/tests/testthat/helper-s3.R

```diff
@@ -0,0 +1,18 @@
+#
+# helper-s3.R
+#
+# Copyright (C) 2022 by RStudio, PBC
+#
+# Unless you have received this program directly from RStudio pursuant
+# to the terms of a commercial license agreement with RStudio, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+#
+
+local_methods <- function(..., .frame = rlang::caller_env()) {
+  rlang::local_bindings(..., .env = rlang::global_env(), .frame = .frame)
```

Comment:
I assume you can remove this since it didn't end up working?

Reply:
oh yes, thanks.